workinggroups/serverless.md

```diff
@@ -0,0 +1,39 @@
+# CNCF Serverless WG Proposal
+
+## TOC Sponsor
+
+Ken Owens
+
+## Objective
+
+Explore the intersection of cloud native and serverless technology.
+
+## Goals and Expected Outcomes
+
+* Produce a whitepaper
+* Produce a serverless landscape
+* Explore specifications for serverless to propose to the CNCF
+* Bring recommendations to the TOC on serverless projects in CNCF
+
+## Non Goals
+
+* Define one serverless project to rule them all
+
+## Initial Interested Parties
+
+* Sarah Allen (Google)
+* Chris Aniszczyk (CNCF)
+* Chad Arimura (Oracle)
+* Ben Browning (Red Hat)
+* Lee Calcote (SolarWinds)
+* Amir Chaudhry (Docker)
+* Doug Davis (IBM)
+* Louis Fourie (Huawei)
+* Antonio Gulli (Google)
+* Yaron Haviv (iguazio)
+* Daniel Krook (IBM)
+* Orit Nissan-Messing (iguazio)
+* Chris Munns (AWS)
+* Ken Owens (Mastercard)
+* Mark Peek (VMWare)
+* Cathy Zhang (Huawei)
```

Comment:
Sometimes companies are listed and sometimes GitHub handles are listed (e.g., on the networking WG proposal). Should this be consistent between proposals? If we are going to pick one I would like to see the GitHub handle but I would prefer to record both. It's nice to see the company at a glance and have a quick way to find a GitHub id for more info.

Reply:
I wonder what qualified a company to be in this list?

Reply:
That was the original group of volunteers. This is the process for companies to be represented in CloudEvents: https://github.com/cloudevents/spec/blob/master/GOVERNANCE.md#voting